pymc3/model_graph.py

```diff
@@ -108,7 +104,15 @@ def _make_node(self, var_name, graph):
         if hasattr(v, 'distribution'):
             distribution = v.distribution.__class__.__name__
         else:
-            distribution = 'Deterministic'
+            is_data = False # type: bool
```

Comment:
From looking at the source code for the `Data` class, you only need to test if `v` is an instance of `SharedVariable`. A deterministic will never be an instance of `SharedVariable` because of the call to `copy` which at most will return a view of a shared variable, which is not an instance of `SharedVariable` itself.

Reply:
@lucianopaz 
Thanks! Fixed that and the test that was failing.  If this is OK with people, we can merge it (although I will clean up the history first).
I would *like* to figure out how to put data that are observations *below* the observed node, instead of above, but I don't sufficiently understand what can be in the `observed` property of a random variable.  Do I need to check to see if it's a `SharedVariable`, and then see if it has a `name` property?  Are those sufficient and necessary conditions?